core/src/main/scala/ai/h2o/sparkling/backend/internal/InternalBackendReader.scala

```diff
@@ -18,13 +18,14 @@
 package ai.h2o.sparkling.backend.internal
 
 import ai.h2o.sparkling.backend.shared.Reader
+import org.apache.spark.h2o.H2OConf
 import water.fvec.{Chunk, Frame, Vec}
 import water.parser.BufferedString
 import water.{DKV, Key}
 
 import scala.language.postfixOps
 
-class InternalBackendReader(override val keyName: String, override val chunkIdx: Int) extends Reader {
+class InternalBackendReader(override val keyName: String, override val chunkIdx: Int, val conf: H2OConf) extends Reader {
```

Comment:
Is the conf argument required?

Reply:
it's reqired by https://github.com/h2oai/sparkling-water/pull/1950/files#diff-14e9f8743d500fef4414bd86722b3e63R27 since `InternalBackendReader` transitively  inherits from Timezone conversions.

Reply:
👍 oki, thanks for explanation!